test/cpp/end2end/xds/xds_end2end_test.cc

```diff
@@ -7160,7 +7154,7 @@ class XdsServerSecurityTest : public XdsEnd2endTest {
     builder.AddCertificateProviderPlugin("file_plugin", "file_watcher",
                                          absl::StrJoin(fields, ",\n"));
     InitClient(builder);
-    CreateBackends(1, /*xds_enabled=*/true);
+    CreateBackends(1, /*xds_enabled=*/true, /*allow_put_requests=*/true);
```

Comment:
I think we need this only for the RBAC tests, not the other security tests, right?

Reply:
Done!